WcaOnRails/config/locales/locales.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+module Locales
+  AVAILABLE = {
+    "en": {
+      "flagId": "gb",
```

Comment:
nit: i think it would fit more with our ruby style for this to be `flag_id` instead of `flagId`. I think the only place we've been doing camelCase is in class and module names.